src/mats_l1_processing/read_parquet_functions.py

```diff
@@ -198,6 +199,7 @@ def read_ccd_data_in_interval(
         filesystem (FileSystem):    Optional. File system to read. If not
                                     specified will assume that path points to
                                     an ordinary directory disk. (Default: None)
+        filter (dict):              an extra filter in form dict {fieldname, [min,max],....}
```

Comment:
Comment has wrong type and does not follow the same conventions as the others. I suggest:
```suggestion
        filter (Optional[dict]):    Extra filters of the form:
                                    `{fieldname1: [min, max], ...}`
```

Reply:
Or something like that.